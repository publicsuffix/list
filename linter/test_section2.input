// test:
// - two ICANN sections

// ===BEGIN ICANN DOMAINS===

example.com

// ===END ICANN DOMAINS===

// ===BEGIN ICANN DOMAINS===
// ===END ICANN DOMAINS===
