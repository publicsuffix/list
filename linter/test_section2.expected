11: warning: 2 ICANN sections found
11: warning: No PRIVATE section found
